documentation/api/io.strimzi.api.kafka.model.KafkaAuthorizationOpa.adoc

```diff
@@ -1,30 +1,30 @@
-To use link:https://www.openpolicyagent.org/[Open Policy Agent^] authorization, set the `type` property in the `authorization` section to the value `opa`.
-The Open Policy Agent authorizer has several configuration options:
+To use link:https://www.openpolicyagent.org/[Open Policy Agent^] authorization, set the `type` property in the `authorization` section to the value `opa`,
```

Comment:
as far as I understood these changes on OPA related stuff are not related to the PR purpose. Can we do that in a different PR?

Reply:
They're just a few formatting changes to make consistent with the other descriptions added to the appendix. So it's sort of related.